src/features/unified-share-modal/SharedLinkSection.js

```diff
@@ -331,7 +331,6 @@ class SharedLinkSection extends React.Component<Props, State> {
                         allowedAccessLevels={allowedAccessLevels}
                         changeAccessLevel={changeSharedLinkAccessLevel}
                         enterpriseName={enterpriseName}
-                        item={item}
```

Comment:
there's a `TODO` in this file around line ~388 to ~406 that uses `isBoxCanvasFile`. Do we still need those changes?

Reply:
Done